app/forms/instance-create.tsx

```diff
@@ -669,6 +748,142 @@ const AdvancedAccordion = ({
+            </TipIcon>
+          </h2>
+          {isFloatingIpAttached && (
+            <MiniTable.Table>
+              <MiniTable.Header>
+                <MiniTable.HeadCell>Name</MiniTable.HeadCell>
+                <MiniTable.HeadCell>IP</MiniTable.HeadCell>
+                {/* For remove button */}
+                <MiniTable.HeadCell className="w-12" />
+              </MiniTable.Header>
+              <MiniTable.Body>
+                {attachedFloatingIpsData.map((item, index) => (
+                  <MiniTable.Row
+                    tabIndex={0}
+                    aria-rowindex={index + 1}
+                    aria-label={`Name: ${item.name}, IP: ${item.ip}`}
+                    key={item.name}
+                  >
+                    <MiniTable.Cell>{item.name}</MiniTable.Cell>
+                    <MiniTable.Cell>{item.ip}</MiniTable.Cell>
+                    <MiniTable.Cell>
+                      <button
+                        onClick={() => {
+                          detachFloatingIp(item.name)
+                        }}
+                      >
+                        <Error16Icon title={`remove ${item.name}`} />
+                      </button>
+                    </MiniTable.Cell>
+                  </MiniTable.Row>
+                ))}
+              </MiniTable.Body>
+            </MiniTable.Table>
+          )}
+          {floatingIpList.items.length === 0 ? (
+            <div className="flex max-w-lg items-center justify-center rounded-lg border p-6 border-default">
+              <EmptyMessage
+                icon={<IpGlobal16Icon />}
+                title="No floating IPs found"
+                body="Create a floating IP to attach it to this instance"
+              />
+            </div>
+          ) : (
+            <div>
+              <Button
+                size="sm"
+                className="shrink-0"
+                disabled={availableFloatingIps.length === 0}
+                disabledReason="No floating IPs available"
+                onClick={addFloatingIpPlaceholder}
+              >
+                Attach floating IP
+              </Button>
+            </div>
+          )}
+
+          {!!attachedFloatingIps.length && (
+            <Modal
+              isOpen={isEmptyFloatingIpPlaceholderPresent}
+              onDismiss={removeFloatingIpPlaceholder}
+              title="Attach floating IP"
+            >
+              <Modal.Body>
+                <Modal.Section>
+                  <Message
+                    variant="info"
+                    content={
+                      <>
+                        This instance will be reachable at{' '}
+                        {selectedFloatingIp ? (
+                          <HL>
+                            {
+                              attachableFloatingIps.find(
+                                (ip) => ip.name === selectedFloatingIp
+                              )?.ip
```

Comment:
Indentation is a bit out of hand here. Might help to pull this out. Try a few things like that.